src/site/content/en/third_party/chrometober-2022/chrometober.njk

```diff
@@ -11,8 +11,7 @@
 		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
 		<link href="https://fonts.googleapis.com/css2?family=Germania+One&amp;family=Nanum+Gothic:wght@400;700&amp;display=swap" rel="stylesheet">
 		<!-- <script is:inline type="module" src="https://flackr.github.io/scroll-timeline/dist/scroll-timeline.js"></script> -->
-	<link rel="stylesheet" href="/third_party/chrometober-2022/assets/index.9ed1d485.css">
-{% include 'partials/analytics.njk' %}
+  	<link rel="stylesheet" href="/third_party/chrometober-2022/assets/index.9ed1d485.css">
```

Comment:
As Chrometober doesn't use the main JavaScript, tracking will stop for this page.

Reply:
Analytics tracking was actually never happening on this page. If you look at what the `analytics.njk` partial was doing (prior to my changes) it was just initializing some variables and not loading the Google Analytics script.

 If you visit the page itself you can verify that nothing is being loaded from or sent to `https://www.google-analytics.com`: https://web.dev/third_party/chrometober-2022/chrometober.html

I think the  fact that this went unnoticed further highlights the need for a central analytics implementation rather than individual pages including (or not including) a partial.

Reply:
This is the directcion this PR takes - to move analytics init to the entrypoint app.js file instead of partial. Another thing is to make all pages use the templates consistently.